core/store/migrations/migrate.go

```diff
@@ -91,6 +92,10 @@ func Migrate(db *gorm.DB) error {
 			ID:      "1565877314",
 			Migrate: migration1565877314.Migrate,
 		},
+		{
+			ID:      "1566915476",
+			Migrate: migration1566915476.Migrate,
+		},
```

Comment:
Is it kosher to insert into this migration array? or is it append-only?

Reply:
It does execute these in order, but it skips played migrations, so unless you're relying on something in the migration after this one it should be fine.